### ISSUE TITLE
[BUG] Errors presented visually are not relayed programmatically.

### ISSUE THREAD

**Issue by KatiePUX** [UKHO/admiralty-design-system#181]
Status Messages
Errors presented visually are not relayed programmatically.

DAC Accessibility Audit outcome: 
[https://ukho.sharepoint.com/sites/EXT_Data_Upload/Shared%20Documents/Forms/AllItems.aspx?OR=Teams%2DHL&CT=1711376954679&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yNDAyMjkyMDYwMiIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D&id=%2Fsites%2FEXT%5FData%5FUpload%2FShared%20Documents%2FGeneral%2FAccessibility%20Audit%2FPrivate%20Beta%2FDAC%20Accessibility%20Report%20WCAG%202%2E2%20UKHO%20UDU%20130324%20Final%20Version%2Epdf&viewid=8b1f0a9e%2D2897%2D4a4d%2D8283%2Dcc660e97e3e4&parent=%2Fsites%2FEXT%5FData%5FUpload%2FShared%20Documents%2FGeneral%2FAccessibility%20Audit%2FPrivate%20Beta](url)

(Pages 44-46)

WCAG Reference:
**4.1.3 Status Messages (Level AA)**
Understanding Status Messages |How to Meet Status Messages
Issue ID: DAC_Status_Messages_01
URL: https://ukhodataupload.admiralty.co.uk/external/survey-details
Page title: Survey Details
Journey: 1.3

Live error messages and the error summary is not adequately relayed to screen reader 
users. This instance can be found when causing errors on the page or attempting to 
continue onto the next stage of the service with required feels left empty. Error messages 
and the error summary can be viewed; however, an adequate role is missing from the 
components to relay this information to screen reader users when they are displayed

**Comment by KatiePUX** [comment]
Is this linked to #161 that @lstevens98 is working on?
